src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/TransactionTest/TransactionEnlistmentTest.cs

```diff
@@ -47,7 +47,6 @@ public static void TestManualEnlistment_Enlist_TxScopeComplete()
             RunTestSet(TestCase_ManualEnlistment_Enlist_TxScopeComplete);
         }
 
-        [SkipOnTargetFramework(TargetFrameworkMonikers.Netcoreapp)]
         [ConditionalFact(typeof(DataTestUtility), nameof(DataTestUtility.AreConnStringsSetup), nameof(DataTestUtility.IsNotAzureServer))]
         public static void TestEnlistmentPrepare_TxScopeComplete()
```

Comment:
I would suggest looking more into whether Enlistment prepared in .NET Core?
The test expects that [Prepare](https://github.com/dotnet/SqlClient/blob/c1984b822f8d31794cd8c85245cc6d7bd1059526/src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/TransactionTest/TransactionEnlistmentTest.cs#L200) is called that forces a rollback and calling `txScope.Complete();` would lead to an exception.

It seems the failing tests are due to no error being thrown.

Reply:
The try/catch and the assert was masking another exception. I reworked it to use Assert.Throws. I forgot that DTC support works on .NET 7 and higher only on windows for now. See https://github.com/dotnet/runtime/issues/715

The remaining work is tracked in https://github.com/dotnet/runtime/issues/71769

I will try to find ways to properly guard the tests there with a conditional fact